src/role_sourcer.js

```diff
@@ -14,25 +14,24 @@
 roles.sourcer = {};
 
 roles.sourcer.settings = {
-  param: ['controller.level'],
+  param: ['energyCapacityAvailable'],
   prefixString: {
-    1: 'MW',
-    3: 'MWC'
+    150: 'MW',
+    600: 'MWC'
   },
   layoutString: {
-    2: 'W',
-    3: 'MW',
-    5: 'MW'
+    0: 'W',
```

Comment:
300 is capacity of spawn. we allways have spawn, so 300 is minimum value of energyCapacityAvailable